classes/PodsInit.php

```diff
@@ -1756,10 +1842,42 @@ public function run() {
 		// Compatibility for Query Monitor conditionals
 		add_filter( 'query_monitor_conditionals', array( $this, 'filter_query_monitor_conditionals' ) );
 
-		// Add WP-CLI commands
+		// Remove Common menus
+		add_action( 'admin_menu', array( $this, 'remove_common_menu' ), 11 );
+		add_action( 'network_admin_menu', array( $this, 'remove_common_network_menu' ), 11 );
+
+		tribe_register_provider( \Pods\REST\V1\Service_Provider::class );
+
+		// Add WP-CLI commands.
 		if ( defined( 'WP_CLI' ) && WP_CLI ) {
-								}
+			//require_once PODS_DIR . 'classes/cli/Pods_CLI_Command.php';
+			//require_once PODS_DIR . 'classes/cli/PodsAPI_CLI_Command.php';
```

Comment:
* :no_entry_sign: No space found before comment text; expected "// require_once PODS_DIR . 'classes/cli/PodsAPI_CLI_Command.php';" but found "//require_once PODS_DIR . 'classes/cli/PodsAPI_CLI_Command.php';"
* :no_entry_sign: Inline comments must end in full-stops, exclamation marks, or question marks

<sub>hash: 8f17be36440a222f0cc5a28241694d10</sub>